src/app/containers/App/App.jsx

```diff
@@ -15,14 +15,19 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
     errorCode,
     route: { pageType },
   } = getRouteProps(routes, location.pathname);
-
   const { pageData, status, error, timeOnServer } = initialData;
 
+  const pageLang = pageData.metadata.language;
```

Comment:
Two points here:
1) Do all pages have a `metadata.language`? 
2) We should have a function with this logic, so we can have it in a separate file in `/App/getVariantOverride.js` and have units test for it. 

```
const getVariantOverride = () =>
...
```
Then we can use it in this `App/App.jsx` file.

Reply:
> Do all pages have a metadata.language?
not all pages would have this, for example most read page data.

> We should have a function with this logic, so we can have it in a separate file in /App/getVariantOverride.js and have units test for it.
good idea, I will apply these changes now